ui/src/app/applications/components/resources.ts

```diff
@@ -11,6 +11,7 @@ export const resourceIcons = new Map<string, string>([
     ['DaemonSet', 'ds'],
     ['Endpoint', 'ep'],
     ['Endpoints', 'ep'],
+    ['EndpointSlice', 'ep'],
```

Comment:
it can be confused, tbh. Because for ep people expect to have endpoints and not slice one

Reply:
Hi @pasha-codefresh 
In this place, I just added another resource-icon mapping to EndpointSlice, as before it does not have an icon and just used ES as default. As mentioned in the issue, he wanted to use the same icon of Endpoint because the behavior is quite similar between them, do you think the change is valid or do you have any comment on how to fix it?

Reply:
I'm actually _okay_ with using `ep`, however to Pasha's point; K8S does differentiate the two.

```shell
$ kubectl api-resources  | grep endpoint
endpoints                           ep           v1                                true         Endpoints
endpointslices                                   discovery.k8s.io/v1               true         EndpointSlice

$ kubectl version 
Client Version: v1.31.0
Kustomize Version: v5.4.2
Server Version: v1.31.0
```

So maybe change it to `eps`?

Reply:
https://github.com/argoproj/argo-cd/pull/19362 has been merged, even though the same concerns apply.

As I understand these comments, ArgoCD will now show `ep` as the label for EndpointSlices? I am glad to get a proper icon now, but this doesn't seem optimal.

Reply:
@JessieTeng89 i still think that it will be confusing to have same icon and same text as endpoint. Appreciate if you can open another PR and change icon. And definitely big thank you for your effort in contribution

Reply:
But from another hand if it didnt have icon at all, it was definitely good PR, i just didnt know about this case

Reply:
Wait.. Will the text/label actually be the same? According to the screenshot in the PR description, the icon has been changed successfully, but the label is still `endpointslice`, which is totally fine in my opinion.